src/Application/Items/Commands/UpgradeUserItemCommand.cs

```diff
@@ -89,6 +89,7 @@ public async Task<Result<UserItemViewModel>> Handle(UpgradeUserItemCommand req,
                 IsBroken = false,
             };
 
+            _db.EquippedItems.RemoveRange(userItemToUpgrade.EquippedItems);
```

Comment:
The EquippedItems are not loaded. Also could you update the tests.

Reply:
Actually we don't want to unequip it but replace it with the upgraded one

Reply:
![image](https://github.com/verdie-g/crpg/assets/47090987/bed1d2cd-39a8-447a-bb35-272f59c0f372)